airbyte-integrations/connectors/destination-gcs/src/main/java/io/airbyte/integrations/destination/gcs/GcsDestination.java

```diff
@@ -90,4 +98,16 @@ public AirbyteMessageConsumer getConsumer(final JsonNode config,
         configuredCatalog);
   }
 
+  /**
+   * The method is used to verify that user set location that match to actual bucket's location if it
+   * already exists
+   */
+  private void assertRegion(final String regionFromConfig, final String actualBucketRegion) {
+    if (!regionFromConfig.equals(actualBucketRegion)
```

Comment:
There is an edge case here with `AUSTRALIA-SOUTHEAST1` or `AUSTRALIA-SOUTHEAST2` and `US`
```
!("AUSTRALIA-SOUTHEAST1".contains("US"))
```
evaluates to `false` and will not trip the if condition

Reply:
Crap, great notice! Thanks! That makes this much more complicated :(

Reply:
Updated, thanks

Reply:
How does this handle dual region buckets? (This is the [list of bucket locations I'm using](https://cloud.google.com/storage/docs/locations)) I am honestly not sure what the behavior is supposed to be but I would assume that `NAM4` which is a "dual region" us bucket, is compatible with `US-WEST1` ?